docs/en/release_notes/release-3.4.md

```diff
@@ -4,6 +4,28 @@ displayed_sidebar: docs
 
 # StarRocks version 3.4
 
+## 3.4.6
+
+Release Date: August 7, 2025
+
+### Improvements
+
+- When exporting data to Parquet files using `INSERT INTO FILES`, you can now specify the Parquet version via the `parquet.version` property to improve compatibility with other tools when reading the exported files. [#60843](https://github.com/StarRocks/starrocks/pull/60843)
```

Comment:
Will it be better to give a link to https://docs.starrocks.io/docs/sql-reference/sql-functions/table-functions/files/#parquetversion?